src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,15 @@
-// Don't forget to import the React library
+import React from 'react';
+import { TodoInfo, Todo } from '../TodoInfo';
 
 // Create a `TodoList` component accepting an array of `preparedTodos` and
 // rendering them as a list
 
-// Add a default export statement for TodoInfo component to use it in the other files
+export const TodoList: React.FC<Todo[]> = (todoList: Todo[]) => {
+  const todoListArray:Todo[] = Object.values(todoList);
+
+  return (
+    <div>
+      { todoListArray.map(todo => <TodoInfo {...todo} />) }
```

Comment:
if previous comment is correct
```suggestion
      { todoList.map(todo => <TodoInfo {...todo} />) }
```